setup.sh

```diff
@@ -100,9 +100,9 @@ function check_dependencies()
     # Installation failed...
```

Comment:
Squash the current third commit with the current first commit of the series (the one that introduced these lines).